v1/calorie_mapper_v1.ipynb

```diff
@@ -226,25 +228,53 @@
    "metadata": {},
    "source": [
     "### Subtask 4\n",
-    "* Text matching to find the first match between the label array and the calorie_counts dataset, identify the matching row"
+    "* Design a database that holds the calorie_counts datasets then find the best match between the label array and the database"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 29,
+   "execution_count": 93,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# read the dataset, clean and arrange data into a dict for quicker lookup.\n",
-    "cal_data = pd.read_csv('../data/calorie_counts.csv').drop(['Serving'], axis=1)\n",
-    "cal_data['Food'] = cal_data['Food'].str.lower()\n",
-    "cal_dict = dict(zip(cal_data['Food'], cal_data['Calories']))\n",
+    "#create the connection to firebase\n",
+    "firebase_config = {'apiKey': \"AIzaSyDLBp2GpdSOd-p16l8ghvSKJ9X008WNAKk\",\n",
+    "  'authDomain': \"calorie-counts-2e344.firebaseapp.com\",\n",
+    "  'projectId': \"calorie-counts-2e344\",\n",
+    "  'storageBucket': \"calorie-counts-2e344.appspot.com\",\n",
+    "  'messagingSenderId': \"711145706388\",\n",
+    "  'appId': \"1:711145706388:web:1cd7d7d8010dc633b3b581\",\n",
+    "  'databaseURL': \"https://calorie-counts-2e344-default-rtdb.europe-west1.firebasedatabase.app/\"\n",
```

Comment:
Let's keep the api keys, ids and url private, maybe follow the same practice as with the google vision api key (for ex: create a local file, add it to .gitignore and read from it here instead of hardcoding the values, share that file separately among the group without making it publicly available) - we probably don't want to go for a secret manager/anything too complicated at this stage but we shouldn't keep the keys public either (even in commit history) :) 

Ref - https://stealthbits.com/blog/how-to-hide-api-keys-github/